packages/features/calendars/weeklyview/components/DateValues/index.tsx

```diff
@@ -12,7 +12,7 @@ export function DateValues({ days, containerNavRef }: Props) {
   return (
     <div
       ref={containerNavRef}
-      className="bg-default dark:bg-muted border-b-subtle sticky top-0 z-30 flex-none border-b sm:pr-8">
+      className="bg-default dark:bg-muted border-b-subtle sticky top-[var(--calendar-dates-sticky-offset,0px)] z-[80] flex-none border-b sm:pr-8">
```

Comment:
Instead of making it always top-0 as a sticky element, I made it dynamic by optionally overriding a css variable, that still defaults to 0.